cache/cache.go

```diff
@@ -48,6 +48,12 @@ type watchCtx struct {
 	onFirstResponse func() // callback to fire once on first upstream response
 }
 
+// batch groups all events that share one ModRevision.
+type batch struct {
```

Comment:
`batch`, the concept you introduced, is not a first-class. It's not standard to etcd API nor any etcd user. It's a structure you introduced for implementation of cache. An implementation detail. This means it should not be exposed externally and should be used in limited scope.

Basically reverse of replacing all mentions of `events` with `batch`.